chain/rpc/src/client.rs

```diff
@@ -192,30 +192,30 @@ impl RetryPolicy<JsonRpcProviderClientError> for SimpleJsonRpcRetryPolicy {
 
         // Retry if a global minimum of number of retries was given and wasn't yet attained
         if self.min_retries.is_some_and(|min| num_retries <= min) {
-            debug!("retrying because {num_retries} is not yet minimum number of retries");
+            debug!(num_retries, min_retries = ?self.min_retries,  "retrying because minimum number of retries not yet reached");
             return RetryAfter(backoff);
         }
 
         match err {
             // Retryable JSON RPC errors are retries with backoff
             JsonRpcProviderClientError::JsonRpcError(e) if self.is_retryable_json_rpc_error(e) => {
-                debug!("encountered retryable JSON RPC error code: {e}");
+                debug!(error = ?e, "encountered retryable JSON RPC error code");
```

Comment:
Better to use `%e` for `Display` formatting when it's an error type. `?e` would be `Debug` formatting, which is more suitable for complex types or for types that don't implement `Display`.

Reply:
thanks, corrected